cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/3.0/casServiceValidationSuccess.jsp

```diff
@@ -49,9 +49,6 @@
                         <cas:${fn:escapeXml(attr.key)}>${fn:escapeXml(attrval)}</cas:${fn:escapeXml(attr.key)}>
                     </c:forEach>
                 </c:forEach>
-                <cas:isFromNewLogin>${fn:escapeXml(isFromNewLogin)}</cas:isFromNewLogin>
```

Comment:
I'd say, yes and no. Changes like those that you propose would require somewhat major changes to the protocol, and would *likely* break those clients that do consume attributes (like the PHP one for example). At the same time, consuming one block for the client is easier than two...and even if you broke it out in 2 different tags, you'd still have to look up attributes by name anyway. So I am not sure I see any practical benefits.